### DISCUSSION TITLE
How to access stand-alone resources from inside a docker-compose resource

### DISCUSSION THREAD

**Discussion by jsimonrichard** [coollabsio/coolify#2925] [Q&A]
I just started using Coolify yesterday, and by-in-large the experience has been great! However, I've had some container networking issues.

I'm trying to access a stand-alone MySQL database from a docker compose resource; I set "Connect to Predefined Network." However, I can't figure out what the host name of the database container is. I've tried the database name (`mysql-database-pco8ckc`) and the container name as reported in the MySQL URL field (`t0swkks`), but neither of those domain names resolve (using `ping`). When I try to access the DB using `mysqladmin`, it hangs forever (or until it hits a timeout).

I looked at the existing docker networks. There's `coolify`, which contains the main Coolify container and all of my stand-alone DBs. Then there's `mogooc`, which contains my docker compose resource and a `coolify-proxy` container. Is the `coolify-proxy` container somehow supposed to give me access to the `coolify` network?

**Comment by credli** [comment]
When you select “Connect to Predefined Network” in Coolify, your service is exposed on Coolify’s external network, making it accessible via http://[hosting-domain-or-ip-address]:3306. To access your MySQL database, you need to ensure that port 3306 is open and accessible from your VPS or dedicated server.

To achieve this, you can either:

1. Expose the port: Open port 3306 in your firewall settings, either through your hosting provider’s control panel or using a tool like ufw on your server if it’s enabled.
2. Use the correct hostname: If you’re accessing the database from another container on the same predefined network, you should be able to use the service name (as defined in your docker-compose.yml) or the container name as the hostname.
3. You can also install phpMyAdmin alongside your MySQL docker instance and use the service name you have set to access it via the web interface.

If you’re still having trouble, check if the coolify-proxy container is correctly routing traffic between networks, or try connecting directly using the container’s IP address.
